environments/lfric/spack.yaml

```diff
@@ -2,7 +2,7 @@ spack:
   specs:
     - xios@2701
     - yaxt@0.11.3 idxtype=long
-    - pfunit@3 max_array_rank=6 +mpi +openmp
+    - pfunit +mpi +openmp
```

Comment:
I have no reason to suspect this won't work fine but just to note we are using pFUnit 4.10.0 at LFRic Apps v2.1/v2.2. This is the latest version on 0.23.0 of Spack so if you're using that you'll get the same. Other versions of Spack might have newer versions, I have no reason to think they won't work but we aren't testing those versions ourselves.

Reply:
We're still on spack 0.22 so 4.10 isn't available, I'll make a note to update spack